src/main/kotlin/systems/ajax/englishstudytelegrambot/controller/WordController.kt

```diff
@@ -0,0 +1,21 @@
+package systems.ajax.englishstudytelegrambot.controller
+
+import org.slf4j.LoggerFactory
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+import systems.ajax.englishstudytelegrambot.service.AdditionalInfoAboutWordService
+
+@RestController
+@RequestMapping("/api")
+class WordController(val additionalInfoAboutWordService: AdditionalInfoAboutWordService){
+
+    val log = LoggerFactory.getLogger(javaClass)
+
+    @GetMapping("/{word}")
+    suspend fun getAdditionalInfoAboutWord(@PathVariable("word") wordSpelling: String){
```

Comment:
why do you need suspend here? and why are you logging service result instead of actually returning it?